recipes/ldas-tools-utilities/meta.yaml

```
@@ -0,0 +1,58 @@
+{% set name = "ldas-tools-utilities" %}
+{% set version = "2.6.4" %}
+
+# dependencies
+{% set ldas_tools_cmake_version = "1.0.4" %}
+{% set ldas_tools_diskcacheapi_version = "2.6.3" %}
+{% set ldas_tools_frameapi_version = "2.6.3" %}
+{% set ldas_tools_ldasgen_version = "1.0.4" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: http://software.ligo.org/lscsoft/source/{{ name }}-{{ version }}.tar.gz
+  sha256: c31b176d02c6647313f3253d14056c41d5f69e204abe75d9480f935d8f2f2d2d
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake >=3.6
+    - pkg-config  # [unix]
+    - make  # [unix]
+  host:
+    - ldas-tools-cmake >={{ ldas_tools_cmake_version }}
+    - ldas-tools-al
+    - ldas-tools-diskcacheapi >={{ ldas_tools_diskcacheapi_version }}
+    - ldas-tools-frameapi >={{ ldas_tools_frameapi_version }}
+    - ldas-tools-framecpp
+    - ldas-tools-ldasgen >={{ ldas_tools_ldasgen_version }}
+  run:
+    - ldas-tools-al
+    - ldas-tools-diskcacheapi >={{ ldas_tools_diskcacheapi_version }}
```

I think all of these run dependencies are being used as shared libraries? If so, you should change them to be:
```suggestion
    - {{ pin_compatible('ldas-tools-diskcacheapi', max_pin='x') }}
```
You might need `x.x` or `x.x.x` depending on the ABI stability you expect.

---

@chrisburr, we are probably going to use `run_exports` in the upstream ldas-tools recipes so that downstream packages don't have to think about it. Is there a good reason to choose one method over the other?

---

If you have `run_exports` you don't need any `run` dependencies as they'll be automatically put there.

The issue with using `ldas-tools-diskcacheapi >=X.Y` instead of `pin_compatible` is that you assume all future versions will be compatible. (`{{ pin_compatible('name', , max_pin='x') }}` is equivalent to `name >=X.Y,<{{X+1}}`)

---

implemented in e810ab2